velodyne_driver/cfg/VelodyneNode.cfg

```diff
@@ -11,5 +11,6 @@ gen = ParameterGenerator()
 gen.add("time_offset", double_t,  1, "A manually calibrated offset (in seconds) to add to the timestamp before publication of a message.",
         0.0, -1.0, 1.0)
 gen.add("enabled", bool_t, 2, "Switch to enable and disable lidar packet consumption", True);
```

Comment:
Maybe make the window_size also a parameter then?

Reply:
I think we get enough control with the tolerance parameter and a fixed window size of 10, my preference would go for less parameters